examples/client/beatBuilder.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<head>
+    <title>Crunker test</title>
+    <script src="https://unpkg.com/crunker@latest/dist/crunker.js"></script>
+    <script type="text/javascript">
+        mergeOneLine = async function(crunker, buffer, times){
+            let audios = times.map(t => crunker.padAudio(buffer, 0, t))
+            return crunker.mergeAudio(audios)
+        }
+        
+        beatBuilder = async function(crunker, beatConf, buffers, beatTiming){
+            let beatLines = await Promise.all(buffers.map((_,i) => {
+                return mergeOneLine(crunker, buffers[i], beatTiming[i].map(bt => bt*beatConf.timing))
+            }))
+            let beat = await crunker.mergeAudio(beatLines)
+            return await mergeOneLine(crunker, beat, Array.from(Array(beatConf.repeats).keys()).map(e => beatConf.delayStart+e*beatConf.beats*beatConf.timing))
+        }
+        
+        window.onload = async function(){
+            let crunker = new Crunker.default({sampleRate:96000});
+            let [hihats] = await crunker.fetchAudio('./drumms_Hi-Hats_Open_Hat.mp3');   //use your own sound here
+            let [dirtBase] = await crunker.fetchAudio('./drumms_dirt_base.mp3');        //use your own sound here
```

Comment:
`suggestion`: Can you please upload these files names with all lowercase spaced hypenated... e.g `./drums-hi-hats-open-hat.mp3`. This helps users run the example to see what the expected input/output is.

For example: https://github.com/jackedgson/crunker/tree/master/examples/server

Reply:
This are files, which I didn't made myself, so I will get into license issues, and this repository too. I already had such an issue with my college. I wish that to nobody!!

I am already searching for better audio, or the last way is to record them myself, but this will need some time

Reply:
Understood, thanks for mentioning.